src/main/java/CMV.java

```diff
@@ -195,7 +196,20 @@ private boolean LIC13() {
    * Computes the LIC 14 condition.
    */
   private boolean LIC14() {
-    // TODO: Implementation.
-    return false;
+    if(points.length < 5 || parameters.AREA2 <= 0) return false;
+
+    boolean greaterThanA1 = false;
+    boolean lessThanA2 = false;
+
+    int ePts = parameters.E_PTS;
+    int fPts = parameters.F_PTS;
+    double AREA1 = parameters.AREA1;
+    double AREA2 = parameters.AREA2;
+
+    for(int Pnt = 0; Pnt < points.length - ePts - fPts; Pnt++){
+      if(Point.triangleAreaFromPoints(points[Pnt], points[Pnt + ePts], points[Pnt + ePts + fPts]) > AREA1) greaterThanA1 = true;
+      if(Point.triangleAreaFromPoints(points[Pnt], points[Pnt + ePts], points[Pnt + ePts + fPts]) < AREA2) lessThanA2 = true;
+    }
```

Comment:
If ePts and fPts is zero, the triangles will be created fro the same points. You are accessing the E_PTS, not the point after it (same for F_PTS). There is also a possible index out of bounds problem.

```java
for (int Pnt = 0; Pnt < points.length - ePts - fPts - 2; Pnt++){
  if(Point.triangleAreaFromPoints(points[Pnt], points[Pnt + ePts + 1], points[Pnt + ePts + 1 + fPts + 1]) > AREA1) greaterThanA1 = true;
  if(Point.triangleAreaFromPoints(points[Pnt], points[Pnt + ePts + 1], points[Pnt + ePts + 1 + fPts + 1]) < AREA2) lessThanA2 = true;
}
```